src/System.Management.Automation/engine/Modules/ModuleCmdletBase.cs

```diff
@@ -2027,12 +2027,73 @@ internal bool LoadModuleManifestData(ExternalScriptInfo scriptInfo, ManifestProc
                 Array.Clear(tmpNestedModules, 0, tmpNestedModules.Length);
             }
 
-            // Set the private data member for the module if the manifest contains
-            // this member
-            object privateData = null;
-            if (data.Contains("PrivateData"))
+            // Set the private data member for the module if the manifest contains this member
+            object privateData = data["PrivateData"];
```

Comment:
`PrivateDate` and `PSData` should be members of `SpecialVariables`

Reply:
But they are not variables ... They are module manifest keys.
We could have a static type called `ModuleManifestKeys`, where we can define all manifest key const strings just like `SpecialVariables`. But we can do it in a separate PR.